app/controllers/katello/api/v2/products_bulk_actions_controller.rb

```diff
@@ -50,6 +50,20 @@ def sync_products
       respond_for_async :resource => task
     end
 
+    api :PUT, "/products/bulk/verify_checksum", N_("Verify checksum for one or more products")
+    param :ids, Array, :desc => N_("List of product ids"), :required => true
+    def verify_checksum_products
+      repairable_products = @products.syncable
+      repairable_roots = RootRepository.where(:product_id => repairable_products).has_url
+
+      repairable_repositories = Katello::Repository.where(:root_id => repairable_roots).in_default_view
```

Comment:
you probably wanna filter to only yum repos

Reply:
So at this point, is pulp_rpm the only content type that will offer the repair api? I wasn't sure if additional content types would implement this?

Reply:
thats my understanding, and pulp2 only supports yum as well